docs/_components/QrngRemoval.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div
+    style="
```

Comment:
Maybe it should stand out just a little more by using a different background? For example like `::: warning`?

Reply:
I'm going off a figma given to me but I will suggest it.

Reply:
It was just a thought - not worth spending a lot of time on